<launch>
  <arg name="UAV_NAME_1" default="$(env UAV_NAME_1)" />
  <arg name="RUN_TYPE" default="$(env RUN_TYPE)" />
  <arg name="UAV_NAME_2" default="$(env UAV_NAME_2)"/>
  <arg name="UAV_NAME_3" default="$(env UAV_NAME_3)" />  
  <node pkg="nodelet" type="nodelet" name="standalone_nodelet_2"  args="manager" output="screen"/> 

  <node pkg="nodelet" type="nodelet" name="FakeOdometryNode" args="load fake_odometry/FakeOdometryNode standalone_nodelet_2" output="screen">
    <param name="UAV_NAME_1" type="string" value="$(arg UAV_NAME_1)"/>
    <param name="UAV_NAME_2" type="string" value="$(arg UAV_NAME_2)"/>
    <param name="UAV_NAME_3" type="string" value="$(arg UAV_NAME_3)"/>
    
    <rosparam command="load" file="$(find fake_odometry)/config/params.yaml"/>
  </node>
</launch>